src/app.rs

```diff
@@ -225,6 +250,8 @@ impl App {
     pub fn save_to<P: AsRef<Path>>(&self, p: P) -> io::Result<()> {
         let acts = self.activities.iter().flat_map(|(_, acts)| acts.iter());
         File::create(p.as_ref()).and_then(|f| store_activities(f, acts))?;
+        File::create(format!("{}-config", p.as_ref().display()))
```

Comment:
wouldn't it be better to save to just `config` instead? Otherwise there will be a different config per month

Reply:
Ye, haven't thought of that really. It was just to make it easier to store the config file in a custom place.
Maybe just store it to the ~/.config location

Reply:
sure, use the [dirs](https://docs.rs/dirs/latest/dirs/) crate